### ISSUE TITLE
To remove the error information

### ISSUE THREAD

**Issue by Fankbai** [jupyterlab/jupyterlab-latex#92]
Hi, 
I didn't notice the requirement of the python version to install the jupyterlab_latex on my python3.5. Failed, even though, some file were changed. Once I start the jupyter, the error jumpts out:
```shell
[W 03:04:58.268 NotebookApp] Error loading server extension jupyterlab_latex
    Traceback (most recent call last):
      File "/usr/local/lib/python3.5/dist-packages/notebook/notebookapp.py", line 1546, in init_server_extensions
        mod = importlib.import_module(modulename)
      File "/usr/lib/python3.5/importlib/__init__.py", line 126, in import_module
        return _bootstrap._gcd_import(name[level:], package, level)
      File "<frozen importlib._bootstrap>", line 986, in _gcd_import
      File "<frozen importlib._bootstrap>", line 969, in _find_and_load
      File "<frozen importlib._bootstrap>", line 956, in _find_and_load_unlocked
    ImportError: No module named 'jupyterlab_latex'
```
anything can be done to remove this info?
Thanks.

**Comment by ian-r-rose** [comment]
Hi @Fankbai, sorry for the slow reply. You should be able to find the jupyter notebook config directory by entering `jupyter serverextension list`. That config probably has a `jupyter_notebook_config.json` file, which will have an entry for `jupyterlab_latex`. If you remove that entry, it should fix the error.

**Comment by bollwyvl** [comment]
Closed due to inactivity (and python 3.5 being EOL)
